polar-core/src/partial/partial.rs

```diff
@@ -998,6 +1240,31 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn test_negate_dot() -> TestResult {
+        let p = Polar::new();
+        p.load_str(r#"f(x) if not (x.y = 1 and x.b = 2);"#)?;
+
+        let mut q = p.new_query_from_term(term!(call!("f", [sym!("x")])), false);
+        assert_partial_expression!(next_binding(&mut q)?, "x", "1 != _this.y or 2 != _this.b");
+        assert_query_done!(q);
+
+        Ok(())
+    }
+
+    #[ignore]
```

Comment:
Is this test intentionally ignored? If so, can we leave a comment and record this TODO somewhere?